examples/Web/Controllers/HomeController.cs

```diff
@@ -39,14 +38,13 @@ public HomeController(KafkaDependentProducer<Null, string> producer, IConfigurat
             // interesting, so we just write frivolous messages to demonstrate how to set things up.
             this.topic = config.GetValue<string>("Kafka:FrivolousTopic");
             this.producer = producer;
-            this.randomNumberGenerator = new Random();
             this.logger = logger;
         }
 
         public async Task<IActionResult> Index()
         {
             // Simulate a complex request handler by delaying a random amount of time.
-            await Task.Delay((int)(this.randomNumberGenerator.NextDouble()*100));
+            await Task.Delay((int)(new Random((int)DateTime.Now.Ticks).NextDouble()*100));

```

Comment:
perhaps make a `static Task RandomDelay() => Task.Delay((int)(new Random((int)DateTime.Now.Ticks).NextDouble()*100))` and then consume as `await RandomDelay()` at 2 call sites ?